src/cls/rbd/cls_rbd_client.cc

```diff
@@ -1249,11 +1306,7 @@ namespace librbd {
       }
 
       bufferlist::iterator iter = out_bl.begin();
-      r = mirror_image_get_finish(&iter, mirror_image);
```

Comment:
Minor: the rationale why I keep this style is for debugging -- it's super easy to set a debug position on the failure path vs a common path where you hope the variable isn't optimized away for a breakpoint condition.